spec/formats/consultation_spec.rb

```diff
@@ -1,15 +1,15 @@
 # frozen_string_literal: true
```

Comment:
This looks good. I think the purpose of this spec is mainly now to test the redirect capability in general, so could we rename it to indicate this?

Reply:
Not sure about that because the [detailed guide spec is exactly the same as this test](https://github.com/alphagov/content-publisher/blob/518e7046b386c6256c10c9287f14ab45fd1c4ead/spec/formats/detailed_guide_spec.rb#L3-L22), so I'm don't know what to do about that one. 

However, maybe we could talk about testing another time? I don't think the current system works very well, but I don't want to derail this PR with that discussion.